tuxemon/core/event/actions/add_monster.py

```diff
@@ -21,6 +21,7 @@
 
 from tuxemon.core import monster
 from tuxemon.core.event.eventaction import EventAction
+from tuxemon.core.event import get_npc
 
 
 class AddMonsterAction(EventAction):
```

Comment:
could you update the doc string for this and the other action?

Reply:
whoops, will do